### ISSUE TITLE
Release edit_json feature.

### ISSUE THREAD

**Issue by goelakash** [weecology/retriever#632]
Currently, the `edit_json` feature has been put on hold due to incomplete testing. Since the main implementation has been merged in `master`, and the feature has been only disable at the CLI, we should refactor the code and add tests to complete this.

@henrykironde @ethanwhite Do you have some suggestions on how to go about this?

**Comment by henrykironde** [comment]
yah sure lets get it that part finished

**Comment by henrykironde** [comment]
I think the tests should be just dry runs. The last time I tried it it failed for some edits

**Comment by goelakash** [comment]
Could you post those examples?

**Comment by henrykironde** [comment]
okey based on what I have searched, we had this `https://github.com/weecology/retriever/pull/596`
I can find the exact PR where we had discussed that issue, however from my recollection we may have solved the issue that was causing the feature to fail a times. So we have added some keys in the json file

```
    values["retriever"] = "True"
    values["script_version"] = 1.0
    values["retriever_minimum_version"] = 2.0

```
  
`values["retriever"] = "True"` helps to identify the retriever script from other json files.

```
   values["script_version"] = 1.0
   values["retriever_minimum_version"] = 2.0
```
Help to keep provenance, an new feature that I am working on

**Comment by henrykironde** [comment]
I suggest creating a PR with the edit_json feature and we can go on and test it again.

**Comment by goelakash** [comment]
Okay, will do.

**Comment by goelakash** [comment]
Fixed by #633
